### ISSUE TITLE
Sync to maven central

### ISSUE THREAD

**Issue by francisdb** [nafg/slick-migration-api#51]
Since jcenter is not part of the default sbt/maven reposities and we don't want our devs using anything that might disappear later.

Setting this up is quite trivial
https://www.jfrog.com/confluence/display/BT/Syncing+with+Third-Party+Platforms

https://stackoverflow.com/questions/54636347/how-to-sync-maven-artifact-from-jcenter-to-maven-central

Workaround to avoid pulling in everything from jcenter:
```scala
resolvers in ThisBuild ++= Seq(
  // we avoid pointing to jcenter and pick specific repos
  // this is for slick-migration-api
  Resolver.bintrayRepo("naftoligug", "maven")
)
```

**Comment by seanwentzel** [comment]
This just became somewhat more urgent since JCenter is going to disappear in a few months https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

**Comment by FlorianKirmaier** [comment]
Yes, that's now important.

**Comment by nafg** [comment]
> Workaround to avoid pulling in everything from jcenter:

Not sure what the point is -- using JCenter as a resolver doesn't imply downloading more

Anyway I've now published 0.8.2 to maven central. Note the `organization` has changed to `io.github.nafg.slick-migration-api`.
